volumina/positionModel.py

```diff
@@ -83,13 +83,13 @@ def __init__(self, parent=None):
         self._time       = 0
         self._channel    = 0
         self._shape5D    = [0,0,0,0,0]
-        
+        self._parent = parent
+
```

Comment:
I think it's preferable not to store `parent` as a member here in python, because it is already stored in the `QObject`, and accessible via the `parent()` accessor function.  (Avoiding this references reduces a cycle in the garbage collector, which is also nice.)